lib/active_model_serializers.rb

```diff
@@ -21,5 +21,6 @@ def self.config
   require 'active_model/serializer/version'
   require 'active_model/serializer'
   require 'active_model/serializable_resource'
+  require 'active_model_serializers/support'
```

Comment:
:scissors: is required in json api/link.  no need to require here, I don't think